doc/ch-image.rst

```diff
@@ -647,10 +676,62 @@ are three basic steps:
      Debian derivatives and :code:`dnf`, :code:`rpm`, or :code:`yum` for
      RPM-based distributions.
 
+:code:`RUN` instructions that *do not* seem to need modification are
+unaffected by this mode.
+
 The details are specific to each distribution. :code:`ch-image` analyzes image
 content (e.g., grepping :code:`/etc/debian_version`) to select a
-configuration; see :code:`lib/fakeroot.py` for details. :code:`ch-image`
-prints exactly what it is doing.
+configuration; see :code:`lib/force.py` for details. :code:`ch-image` prints
+exactly what it is doing.
+
+.. warning::
+
+   Because of :code:`fakeroot` mode’s complexity, we plan to remove it if
+   :code:`seccomp` mode performs well enough. If you have a situation where
+   :code:`fakeroot` mode works and :code:`seccomp` does not, please let us
+   know.
+
+Workaround mode :code:`seccomp` (default)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This mode uses the kernel’s :code:`seccomp(2)` system call filtering to
+intercept certain privileged system calls, do absolutely nothing, and return
+success to the program.
+
+The quashed system calls are: :code:`capset(2)`; :code:`chown(2)` and friends;
+:code:`mknod(2)` and :code:`mknodat(2)`; and :code:`setuid(2)`,
+:code:`setgid(2)`, and :code:`setgroups(2)` along with the other system calls
+that change user or group.
+
+The advantages of this approach is that it’s much simpler, it’s faster, it’s
+completely agnostic to libc, and it’s mostly agnostic to distribution. The
+disadvantage is that it’s a very lazy liar; even the most cursory consistency
+checks will fail, e.g., :code:`getuid(2)` after :code:`setuid(2)`.
+
+While this mode does not provide consistency, it does offer a hook to help
+prevent programs asking for consistency. For example, :code:`apt-get -o
+APT::Sandbox::User=root` will prevent :code:`apt-get` from attempting to drop
+privileges, which `it verifies
+<https://salsa.debian.org/apt-team/apt/-/blob/cacdb549/apt-pkg/contrib/fileutl.cc#L3343>`_,
+exiting with failure if the correct IDs are not found (which they won’t be
+under this approach). This can be expressed with
+:code:`--force-cmd=apt-get,-o,APT::Sandbox::User=root`, though this particular
```

Comment:
I totally agree with @mphinney1100's previous (resolved) criticism above, in the help section for `--force-cmd`.  It didn't click with me what it was for...until I got here!  Once I understood `apt`'s validation procedure and required circumvention, it all suddenly made sense!

Might be helpful to add "`Consult the "Workaround mode seccomp" section below for details on why this can be useful (and may be necessary).`"  Maybe even a link to this section, if that's feasible/practical.